Feat: 클립 관리 도메인 기능 구현 완료 #71

kamillee0918 opened the pull request:
## ⭐Key Changes
> 핵심 변경 사항에 대해서 적어주세요.
1. 기존 ‘클립’ 관련 도메인과의 충돌을 피하기 위해 별도의 ‘클립 관리’ 도메인을 작성
2. “클립 추가” 엔드포인트 호출 시, file 테이블의 ‘file_type', ‘file_name’, ‘file_url’에 각각 clip, fileName, clip/{clipId}/{fileNameWithExtension} 와 같이 삽입되도록 로직 수정
3. 테스트 코드 작성 완료

<br>

## 📌Issue
> 닫을 issue 번호 : resolved #71

<br>

## 🩼Branch
> 반영 branch feat-71 -> dev

<br>

## 👪To Reviewers
> 맞게 작성한 것인지, 자신이 없어용... 기능 단위로 구현을 했어야 했는데..

kamillee0918 committed:
Feat: 클립 관리 도메인 기능 구현 완료 #71

### 구현 사항
1. 기존 ‘클립’ 관련 도메인과의 충돌을 피하기 위해 별도의 ‘클립 관리’ 도메인을 작성
2. “클립 추가” 엔드포인트 호출 시, file 테이블의 ‘file_type', ‘file_name’, ‘file_url’에 각각 clip, fileName, clip/{clipId}/{fileNameWithExtension} 와 같이 삽입되도록 로직 수정
3. 테스트 코드 작성 완료

Hexeong commented on src/main/java/com/noblesse/backend/clip/service/ClipReportService.java:
```diff
@@ -11,31 +13,49 @@
+//                clipReportRegistRequestDTO.getUserId(),
+//                clipReportRegistRequestDTO.getClipId()
+//        ));
+//    }
```
없애야 할 내용이면 주석 삭제해주세요:)

kamillee0918 replied:
주석 지울게용..!

kamillee0918 replied:
`ClipReport` 클래스에서 `Clip` 클래스와의 매핑을 설정한 이유는 딱히 없어요.. 가능하면 내일(22일) 이후에 수정할 수 있을 것 같아요..

Hexeong replied:
넹넹

Hexeong reviewed (commented):
연관관계 매핑하신 이유가 따로 있나용

kamillee0918 committed:
Style: ClipReportService 클래스 불필요한 주석 삭제 #71

### 변경 사항
1. 불필요한 주석 삭제
2. ClipReport 클래스에서 Clip과 매핑한 이유는 딱히 없어요.. 내일(22일) 이후에 매핑 관계를 다시 수정해서 Push할게요...

Hexeong commented on src/main/java/com/noblesse/backend/clip/service/ClipService.java:
```diff
@@ -25,15 +29,42 @@ public List<Clip> findAll() {
         return clipRepository.findAll();
     }
 
+//    @Transactional
```
요기도 있어요!

kamillee0918 committed:
Style: 일부 클래스 불필요한 주석 삭제 #71

### 변경 사항
1. 불필요한 주석 삭제

Hexeong reviewed (approved):
굿굿
